### DISCUSSION TITLE
components/ui directory

### DISCUSSION THREAD

**Discussion by TheBinaryGuy** [cosscom/coss#122] [Q&A] [answered]
We already use a lot of Shadcn UI components, if the component already exists in the `components/ui` directory can we reuse those or are your components inside `components/ui` different?

**Comment by pasqualevitiello** [comment, marked as answer]
Sure! Origin UI is compatible with Shadcn. However, note that components which are not supported by Shadcn (eg. vertical sliders) may not render properly. In such cases, you may need to use our `ui` core components.
